.github/workflows/upd-xna-client.yml

```diff
@@ -0,0 +1,161 @@
+# To be able use this workflow you must ensure that your repository have enable write and right access for GitHub Actions.
+#
+# GUI Reference to enable access:
+# Repository --> Actions --> General --> Workflow permissions --> Read and write permissions
+#                                                            \--> Allow Github Actions to create and approve pull requests
+
+name: Update XNA CnCNet Client
+
+on:
+  workflow_dispatch:
+  schedule:
+  - cron: "0 0 * * *"
+
+env:
+  REPO_NAME: cncnet-client-mod-base
+  CLIENT_PATH: ClientFiles
```

Comment:
I do think the comment mentioning that you must not add a trailing slash (and what type of slash to use if you need to do nested folders) has to be placed here.

Reply:
Done